services/dags/airnow_etl.py

```diff
@@ -0,0 +1,161 @@
+"""
+airnow etl functions
+
+for initial setup:
+    1. run airnow_ingest_station_data dag
+    2. run airnow_etl dag (the dag in this file)
+    3. after 1 hr run load prod dag
+"""
+from datetime import datetime as dt, timedelta
+import os
+
+import pandas as pd
+import numpy as np
+import requests
+from airflow.decorators import dag, task
+from airflow.providers.postgres.operators.postgres import PostgresOperator
+from airflow.providers.postgres.hooks.postgres import PostgresHook
+
```

Comment:
in vscode you can right click in the editor, and there's an option for "sort imports"...which is nice to keep imports alphabetized and easier to read